scripts/setup-github-secrets.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# Script to set up GitHub organization-level secrets for Lambda pipeline execution
```

Comment:
Might be preferable to do this per-repo so if you eg add an external collaborator on a random other repo it doesn't immediately give access to the same secrets.